.dockerfiles/ort.Dockerfile

```diff
@@ -26,6 +26,8 @@ ENV PATH="$HOME/.cargo/bin:$PATH"
 
 RUN npm install -g yarn
 
+RUN wget -qO- https://get.pnpm.io/install.sh | ENV="$HOME/.bashrc" SHELL="$(which bash)" bash -
```

Comment:
Not sure why but pnpm was still missing from the PATH in the container. For now I just added an installation step in workflow on my other PR 😋